packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -1,30 +1,21 @@
 <template>
 	<div :class="formGroupStyle">
-		<div
-			:class="decreaseButtonStyle"
-			@click="setCurrentValue(currentValue - step)"
-		>
+		<div :class="decreaseButtonStyle" @click="decrementValue">
 			<i class="yoco">minus</i>
 		</div>
 		<input
 			ref="inputNumber"
 			:class="inputStyle"
-			type="number"
 			:disabled="disabled"
 			:value="currentValue"
-			:max="max"
-			:min="min"
 			@input="setCurrentValue($event.target.value)"
```

Comment:
hmm so i was wondering why we don't have this instead of this custom component then i remembered we wanted the yoco custom plus and minus (works consistently across web clients since this was a bug before) but to see that we had both the type=number, min/max logic AND the divs with the whole decrement/increment control :D wow

Reply:
i returned this back - i thought, but if not i should - 
this gives us the standard html invlaid if the user types in a value that is too high or low.